mii/models/score/generate.py

```diff
@@ -9,33 +9,39 @@
 from mii.constants import DeploymentType
 
 
-def create_score_file(deployment_name,
+def create_score_file(deployment_tag,
                       deployment_type,
-                      task,
-                      model_name,
-                      ds_optimize,
-                      ds_zero,
-                      ds_config,
-                      mii_config,
+                      deployments,
                       model_path,
                       lb_config):
+
     config_dict = {}
-    config_dict[mii.constants.DEPLOYMENT_NAME_KEY] = deployment_name
-    config_dict[mii.constants.TASK_NAME_KEY] = mii.utils.get_task_name(task)
-    config_dict[mii.constants.MODEL_NAME_KEY] = model_name
-    config_dict[mii.constants.ENABLE_DEEPSPEED_KEY] = ds_optimize
-    config_dict[mii.constants.MII_CONFIGS_KEY] = mii_config.dict()
-    config_dict[mii.constants.ENABLE_DEEPSPEED_ZERO_KEY] = ds_zero
-    config_dict[mii.constants.DEEPSPEED_CONFIG_KEY] = ds_config
     config_dict[mii.constants.MODEL_PATH_KEY] = model_path
+    config_dict[mii.constants.DEPLOYMENT_TAG_KEY] = deployment_tag
+    for deployment in deployments:
+        config_dict[deployment.deployment_name] = {}
+        config_dict[deployment.deployment_name][
+            mii.constants.DEPLOYMENT_NAME_KEY] = deployment.deployment_name
+        config_dict[deployment.deployment_name][
+            mii.constants.TASK_NAME_KEY] = mii.utils.get_task_name(deployment.task)
+        config_dict[deployment.deployment_name][
+            mii.constants.MODEL_NAME_KEY] = deployment.model
+        config_dict[deployment.deployment_name][
+            mii.constants.ENABLE_DEEPSPEED_KEY] = deployment.enable_deepspeed
+        config_dict[deployment.deployment_name][
+            mii.constants.MII_CONFIGS_KEY] = deployment.mii_config.dict()
+        config_dict[deployment.deployment_name][
+            mii.constants.ENABLE_DEEPSPEED_ZERO_KEY] = deployment.enable_zero
+        config_dict[deployment.deployment_name][
+            mii.constants.DEEPSPEED_CONFIG_KEY] = deployment.ds_config
```

Comment:
```suggestion
        deployment_config = {
            mii.constants.DEPLOYMENT_NAME_KEY: deployment.deployment_name,
            mii.constants.TASK_NAME_KEY: mii.utils.get_task_name(deployment.task),
            mii.constants.MODEL_NAME_KEY: deployment.model,
            mii.constants.ENABLE_DEEPSPEED_KEY: deployment.enable_deepspeed,
            mii.constants.MII_CONFIGS_KEY: deployment.mii_config.dict(),
            mii.constants.ENABLE_DEEPSPEED_ZERO_KEY: deployment.enable_zero,
            mii.constants.DEEPSPEED_CONFIG_KEY: deployment.ds_config,
        }
        config_dict[deployment.deployment_name] = deployment_config
```

We can make this cleaner and easier to read